### PR TITLE
Shareable playground URL

### LINKED ISSUE #697: Make playground url shareable
Make the playground url shareable with populated fields using base64 https://snapshot.org/#/playground/delegation?query=$base64object 

The query can use the same format as a strategy example see https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/api/examples.json

We shouldn't add a new dependency for base64, browser support it.

### PR DESCRIPTION
**Pull request by RakeshUP**
Fixes #697.

Changes proposed in this pull request:
- Adds a button below "Run/Play" button in playground actions with text "copyLink"
- On click copies the encoded form values as query params to the clipboard
- During component setup, loads playground values from query params if present

### CODE REVIEW

**Comment by samuveth** [comment]
@RakeshUP I tested it and it only seems to be populating the strategy params. We also want, Network, Snapshot and Addresses

**Comment by RakeshUP** [comment]
Hey @samuveth, I just made a change for populating Snapshot. And Network, Addresses seem to be working fine for me. I tested it multiple times. I'll debug again if it's still not working for you.

**Comment by samuveth** [comment]
@RakeshUP works well!
@bonustrack do we want to URL in the address bar also, or is the "Copy link" enough?

File: src/views/Playground.vue [hunk 1]
```diff
@@ -15,11 +17,37 @@ const defaultParams = {
...
 const { strategies } = useApp();
+const { notify } = useNotifications();
+const { t } = useI18n();
 
 let provider;
 
 const strategy = computed(() => strategies.value[route.params.name]);
-const strategyExample = computed(() => strategy.value.examples?.[0]);
+const strategyExample = computed(() => {
+  const queryParamsIndex = window.location.hash.indexOf('?query=');
```
Comment (bonustrack) [inline src/views/Playground.vue:27]: You shouldn't use window location here, you can get the url query params with Vue router directly

**Review by bonustrack — COMMENTED** [review]
Nice! Is it possible to update directly the url when you are changing the params too?

**Comment by RakeshUP** [comment]
I have made the changes to update the URL on editing the params. Because of the previous router behavior, it was always scrolling to top on editing(changing any param would edit the URL causing a re-render)
So I made some changes to the router configuration's scrollBehavior
@bonustrack

**Review by samuveth — APPROVED** [review]
Works well!

**Review by bonustrack — APPROVED** [review]
Awesome!

**Comment by ChaituVR** [comment]
Previously, when we change the network, the block number field below it was changing to the network selected, but now it is not

Example:
- Go to https://bafybeigwl42c5h2rnjd4ek5okmm3wtrbtntriwfbm4jsndfqcs3rawkaiq.on.fleek.co/#/strategy/erc20-balance-of
- Change network to binance network
- Snapshot block number should automatically update to BSC latest block number

**Comment by RakeshUP** [comment]
I've opened a new PR to fix the above issue. #793 
Sorry I missed that during testing

**Pull request merged**